kubernetes/samples/scripts/create-rcu-schema/drop-rcu-schema.sh

```diff
@@ -12,16 +12,14 @@ source ${scriptDir}/../common/utility.sh
 function usage {
   echo "usage: ${script} -s <schemaPrefix> -d <dburl> -n <namespace> -q <sysPassword> -r <schemaPassword> [-h]"
   echo "  -s RCU Schema Prefix (required)"
+  echo "  -q password for database SYSDBA user. (required)"
+  echo "  -r password for all schema owner (regular user). (required)"
   echo "  -t RCU Schema Type (optional)"
   echo "      (supported values: fmw(default), soa, osb, soaosb, soaess, soaessosb) "
   echo "  -d Oracle Database URL (optional)"
   echo "      (default: oracle-db.default.svc.cluster.local:1521/devpdb.k8s) "
   echo "  -n Namespace where RCU pod is deployed (optional)"
   echo "      (default: default) "
-  echo "  -q password for database SYSDBA user. (optional)"
-  echo "      (default: Oradoc_db1)"
```

Comment:
Isn't -q still supported and now required?  If so, please keep line 21,  and change "optional" to "required".

Reply:
Yes, I moved the -q and -r options to the top to keep the required switches together.